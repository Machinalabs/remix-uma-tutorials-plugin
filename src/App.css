html {
    zoom: 88%;
}
body, html, #root, .container-fluid, .row.full-height {
    height: 100%;
}

div.red {
    color: red;
    font-size: 0.75em;
}

input.form-control.form-control-sm::-webkit-input-placeholder { /* Chrome/Opera/Safari */
    color: gray;
}

input.form-control.form-control-sm::-moz-placeholder { /* Firefox 19+ */
    color: gray;
}

input.form-control.form-control-sm:-ms-input-placeholder { /* IE 10+ */
    color: gray;
}
 
input.form-control.form-control-sm:-moz-placeholder { /* Firefox 18- */
    color: gray;
}

.borderBottomExceptLast{
    border-bottom: 1px solid rgba(236, 233, 233, 0.125)
}

.borderBottomExceptLast:last-child{
    border-bottom: none;
}