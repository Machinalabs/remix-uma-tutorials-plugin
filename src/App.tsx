import React from "react"

import { Routes } from "./routes"
import { RemixProvider } from "./hooks"

// TODO Remove
import "bootstrap/dist/css/bootstrap.min.css"

import "./App.css"

const App = () => {
  return (
    <RemixProvider>
      <Routes />
    </RemixProvider>
  )
}

export default App
