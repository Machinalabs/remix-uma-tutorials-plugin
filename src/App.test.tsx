import { render } from "@testing-library/react"

import App from "./App"

describe("App tests", () => {
  test("snapshot", () => {
    const { container } = render(<App />)

    expect(container).toMatchSnapshot()
  })
})
