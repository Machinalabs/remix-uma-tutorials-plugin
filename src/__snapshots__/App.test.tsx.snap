// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests snapshot 1`] = `
<div>
  <div>
    <div>
      Home View
    </div>
  </div>
</div>
`;
